bundles/AdminBundle/Controller/Admin/TagsController.php

```diff
@@ -95,6 +95,43 @@ public function updateAction(Request $request)
         }
     }
 
+    /**
+     * @param Request $request
+     * @Route("/update-translation-for-tag")
+     */
+    public function updateTranslationForTagAction(Request $request){
+        $success = false;
+
+        if($tag = Tag::getById($request->get('id'))){
+            $translations = $tag->getTranslations();
+            $translations[$request->get('language')] = $request->get('translation');
+
+            $tag->setTranslations(serialize($translations));
```

Comment:
I think the serializing should be done in the setter, as the unserializing is done in the getter, too.

```suggestion
            $tag->setTranslations($translations);
```